install_radar_deps.py

```diff
@@ -194,71 +198,73 @@ def install_protobuf():
     """
 
     proto_cmd = "cd ${IDIR};" \
-    "git clone https://github.com/google/protobuf.git;" \
-    "cd protobuf || exit;" \
-    "./autogen.sh;" \
-    "./configure;" \
-    "make -j${CORES};" \
-    "make -j${CORES} check;" \
-    "make install;" \
-    "ldconfig;"
+                "git clone https://github.com/protocolbuffers/protobuf.git;" \
+                "cd protobuf || exit;" \
+                "./autogen.sh;" \
```

Comment:
```suggestion
                "git checkout v3.19.4;" \        # This is to avoid builder.py errors
                "git submodule init && git submodule update;" \  # For googletest library
                "./autogen.sh;" \
```